cmd/kbn/config.go

```diff
@@ -219,24 +219,51 @@ func setAuthorizedNodes(ctx *cli.Context, cfg *bootnodeConfig) {
 // setHTTP creates the HTTP RPC listener interface string from the set
 // command line flags, returning empty if the HTTP endpoint is disabled.
 func setHTTP(ctx *cli.Context, cfg *bootnodeConfig) {
-	if ctx.GlobalBool(utils.RPCEnabledFlag.Name) && cfg.HTTPHost == "" {
+	if ctx.GlobalBool(utils.HTTPEnabledFlag.Name) && cfg.HTTPHost == "" {
```

Comment:
I put comments on another PR, but do you have any better way to check legacy flags?

Reply:
I reflected it in this PR too! 
Thanks JK!